src/k8s/components/indexed-search/statefulset.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: apps/v1
```

Comment:
Did you mean to commit this?

Reply:
oh no, good catch

Reply:
i use the source YAML files side by side in a split editor when i work on a component